docker/models/images.py

```diff
@@ -395,12 +395,12 @@ def load(self, data):
 
         return [self.get(i) for i in images]
 
-    def pull(self, repository, tag=None, **kwargs):
+    def pull(self, repository, tag=None, all_tags=False, **kwargs):
         """
         Pull an image of the given name and return it. Similar to the
         ``docker pull`` command.
-        If no tag is specified, all tags from that repository will be
-        pulled.
+        If ``all_tags`` is set, the ``tag`` parameter is ignored and all image
+        tags will be pulled.
```

Comment:
Even though it's intuitive, it's probably good to add explicit documentation that if `tag` is `None` or empty, then it will be set to `"latest"`. Also in the `api.image` module.